src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Encapsulates the idea of a task, set as abstract
+ * to prevent instantiation
+ */
+package duke.task;
+
+public abstract class Task {
+
+    // String constants for labelling state of task
+    protected static final String CROSS = "✗";
+    protected static final String CHECK = "✓";
+
+    // instance fields
+    protected String desc; // description of task
+    protected boolean isDone; // state of task
+
+    // constructors
+    Task(String desc) {
```

Comment:
Another nitpick! There were certain groups of methods that were separated by an empty row and some there were not, such as these constructors, even though the methods were part of a certain category. Perhaps this could be standardized?